src/jimp2/wireworld/z8/datamangment/FactoryOfCustomElements.java

```diff
@@ -30,6 +111,57 @@ private void initalizeElements() {
 
     public void saveNewCustomElement(World world, Point startingPoint) {
 
+        //do usunięcia później
+        //TODO
+        startingPoint = new Point(0,0);
+
+        final String CustomName = "CustomElement";
```

Comment:
O losie zapomniałem o tym - trzeba wyświetlić jakieś okienko tekstowe gdzie user wpisze sobie nazwę pod jaką chce zapisać custom element. Na 90% nie powinno być to trudne - coś na zasadzie tych okienek komunikujących błędy.